admin/installation.rst

```diff
@@ -376,9 +376,9 @@ Provided driver packages:
 
   * netxms-dbdrv-pgsql - PostgreSQL driver
   * netxms-dbdrv-mariadb - Mariadb driver
-  * netxms-dbdrv-mysql - MySQL driver (not built for Ubuntu 20 / Mint 20)
+  * netxms-dbdrv-mysql - MySQL driver, currently under development (not built for Ubuntu 20 / Mint 20)
   * netxms-dbdrv-odbc - unixODBC driver (can be used with DB/2 and Microsoft SQL)
-  * netxms-dbdrv-oracle - Oracle driver
+  * netxms-dbdrv-oracle - Oracle driver, currently under development ( requires Oracle client installation )
```

Comment:
it's currently built for all available platforms -- amd64 version of the oracle client is still unavailable